iis-wf/iis-wf-affmatching/src/test/resources/data/expectedOutput/matchedAffiliations.json

```diff
@@ -10,3 +10,15 @@
     "organizationId":"20|UW",
     "matchStrength":0.9677419
 }
+{
+    "documentId":"DOC_3",
+    "position":1,
+    "organizationId":"20|UCB",
+    "matchStrength":0.19354838
```

Comment:
Expected `matchStrenght` is pretty low - does it mean we do not trust docOrg matcher too much? I guess we are going to adjust it in future.